src/resources/postcss/views/_list.pcss

```diff
@@ -38,15 +38,22 @@
 	}
 
 	.tribe-events-calendar-list__event--title {
+		color: var(--color-text-primary);
 		font-size: var(--font-size-7);
 		font-weight: var(--font-weight-bold);
+		line-height: var(--line-height-1);
 		margin-bottom: var(--spacer-1);
 		padding: 0;
 
 		@media (--viewport-medium) {
 			font-size: var(--font-size-7);
 			margin-bottom: var(--spacer-2);
 		}
+
+		a {
```

Comment:
could we add a class to this link and target the class only rather than increasing the specificity with nested selectors?